build/develop-env/lsstsqre/Dockerfile

```diff
@@ -59,7 +59,20 @@ RUN yum -y --enablerepo=extras install epel-release \
   tcpdump \
   graphviz* \
   tzdata \
-  libgphoto2-devel
+  gcc \
```

Comment:
I really hate to have to install these libraries (from gcc to make), because they are already available in the conda environment. But I guess you library further ahead won't compile with conda environment alone... :(

Reply:
Correct. When I try to use the gcc included in conda, it fails to locate some of the system headers, like `usb.h`.

Reply:
As a matter of fact, it turns out that all packages, except for make, already were installed in the base image so I removed them, except for make.